src/Util/PhpstormHelper.php

```diff
@@ -48,8 +48,11 @@ public static function handleArgvFromPhpstorm(array &$argv, string $paratestBina
     /** @param  array<int, string> $argv */
     private static function getArgvKeyFor(array $argv, string $searchFor): int
     {
+        $searchForForwardSlash = str_replace('\\', '/', $searchFor);
+        $searchForBackwardSlash = str_replace('/', '\\', $searchFor);
```

Comment:
```suggestion
        $searchFor = str_replace('/', DIRECTORY_SEPARATOR, $searchFor);
```
Instead of a double search, what about a single search with the os-aware `DIRECTORY_SEPARATOR` constant?

Reply:
Unfortunately, both forward and backward slashes can be used within one single command line on Windows, as you can see in the example I provided above. The command line contains:
- `C:\repos\some_project\vendor\bin\paratest_for_phpstorm` and
- `C:/repos/some_project/vendor/phpunit/phpunit/phpunit`. 

The `getArgvKeyFor()` method must support both types. Therefore, it cannot rely on `DIRECTORY_SEPARATOR` as a replacement in `$searchFor` because `$argv` might contain non-OS-standard slashes. Users have the "freedom of choice" to use any type of slash when providing the configuration path.

P.S. Why did I use `str_replace()` twice? On Windows, a single path can contain both types of slashes at the same time and still be valid, e.g., `C:\repos\some_project\vendor\bin/paratest_for_phpstorm`. Using `str_replace()` twice normalizes all paths to contain only one type of slash for easier comparison.